### PR TITLE
media: Apply coding rule using clang-format

### PR DESCRIPTION
**Pull request by junmin-kim**
Apply coding rule using .clang-format in TizenRT

### CODE REVIEW

**Comment by seinfra** [comment]
Target : [a0a68a038c224c3a60734faf6cf11ff824427e1b] - Code Rule Check (C++) OK.

**Comment by seinfra** [comment]
Target : [a0a68a038c224c3a60734faf6cf11ff824427e1b] - Code Rule Check Result:
framework/include/media/BufferOutputDataSource.h:45: ERROR: [SPC_M_OPR] spaces prohibited around that ':' (ctx:WxW)
framework/include/media/BufferOutputDataSource.h:84: ERROR: [SPC_M_OPR] spaces required around that '=' (ctx:VxV)
framework/include/media/DataSource.h:78: ERROR: [SPC_M_OPR] spaces required around that '=' (ctx:VxV)
framework/include/media/FileInputDataSource.h:42: ERROR: [SPC_M_OPR] spaces prohibited around that ':' (ctx:WxW)
framework/include/media/FileInputDataSource.h:63: ERROR: [SPC_M_OPR] spaces required around that ':' (ctx:OxV)
framework/include/media/FileInputDataSource.h:75: ERROR: [SPC_M_OPR] spaces required around that '=' (ctx:VxV)
framework/include/media/FileInputDataSource.h:123: ERROR: [SPC_M_OPR] spaces required around that ':' (ctx:OxV)
framework/include/media/FileOutputDataSource.h:42: ERROR: [SPC_M_OPR] spaces prohibited around that ':' (ctx:WxW)
framework/include/media/FileOutputDataSource.h:58: ERROR: [SPC_M_OPR] spaces required around that ':' (ctx:OxV)
framework/include/media/FileOutputDataSource.h:69: ERROR: [SPC_M_OPR] spaces required around that ':' (ctx:OxV)
framework/include/media/FileOutputDataSource.h:81: ERROR: [SPC_M_OPR] spaces required around that '=' (ctx:VxV)
framework/include/media/FileOutputDataSource.h:123: ERROR: [SPC_M_OPR] spaces required around that ':' (ctx:OxV)
framework/include/media/FocusChangeListener.h:40: ERROR: [SPC_M_KWD] space required before the open brace '{'
framework/include/media/InputDataSource.h:45: ERROR: [SPC_M_OPR] spaces prohibited around that ':' (ctx:WxW)
framework/include/media/InputDataSource.h:65: ERROR: [SPC_M_OPR] spaces required around that '=' (ctx:VxV)
framework/include/media/InputDataSource.h:88: ERROR: [SPC_M_OPR] spaces required around that ':' (ctx:VxO)
framework/include/media/InputDataSource.h:88: ERROR: [SPC_M_OPR] spaces required around that ':' (ctx:OxV)
framework/include/media/InputDataSource.h:88: ERROR: [SPC_M_OPR] spaces required around that '<' (ctx:VxV)
framework/include/media/InputDataSource.h:88: ERROR: [SPC_M_OPR] spaces required around that '>' (ctx:VxW)
framework/include/media/InputDataSource.h:89: ERROR: [SPC_M_OPR] spaces required around that ':' (ctx:OxV)
framework/include/media/InputDataSource.h:89: ERROR: [SPC_M_OPR] spaces required around that '<' (ctx:VxV)
framework/include/media/InputDataSource.h:89: ERROR: [SPC_M_OPR] spaces required around that '>' (ctx:VxW)
framework/include/media/InputDataSource.h:96: ERROR: [SPC_M_OPR] spaces required around that ':' (ctx:OxV)
framework/include/media/InputDataSource.h:96: ERROR: [SPC_M_OPR] spaces required around that '<' (ctx:VxV)
framework/include/media/InputDataSource.h:96: ERROR: [SPC_M_OPR] spaces required around that '>' (ctx:VxW)
framework/include/media/MediaPlayer.h:167: ERROR: [SPC_M_OPR] spaces required around that ':' (ctx:VxO)
framework/include/media/MediaPlayer.h:167: ERROR: [SPC_M_OPR] spaces required around that ':' (ctx:OxV)
framework/include/media/MediaPlayer.h:167: ERROR: [SPC_M_OPR] spaces required around that '<' (ctx:VxV)
framework/include/media/MediaPlayer.h:167: ERROR: [SPC_M_OPR] spaces required around that ':' (ctx:VxO)
framework/include/media/MediaPlayer.h:167: ERROR: [SPC_M_OPR] spaces required around that ':' (ctx:OxV)
framework/include/media/MediaPlayer.h:167: ERROR: [SPC_M_OPR] spaces required around that '>' (ctx:VxB)
framework/include/media/MediaPlayer.h:178: ERROR: [SPC_M_OPR] spaces required around that ':' (ctx:VxO)
framework/include/media/MediaPlayer.h:178: ERROR: [SPC_M_OPR] spaces required around that ':' (ctx:OxV)
framework/include/media/MediaPlayer.h:178: ERROR: [SPC_M_OPR] spaces required around that '<' (ctx:VxV)
framework/include/media/MediaPlayer.h:178: ERROR: [SPC_M_OPR] spaces required around that '>' (ctx:VxB)
framework/include/media/MediaPlayer.h:181: ERROR: [SPC_M_OPR] spaces required around that ':' (ctx:OxV)
framework/include/media/MediaPlayer.h:181: ERROR: [SPC_M_OPR] spaces required around that '<' (ctx:VxV)
framework/include/media/MediaPlayer.h:181: ERROR: [SPC_M_OPR] spaces required around that '>' (ctx:VxW)
framework/include/media/MediaRecorder.h:169: ERROR: [SPC_M_OPR] spaces required around that ':' (ctx:VxO)
framework/include/media/MediaRecorder.h:169: ERROR: [SPC_M_OPR] spaces required around that ':' (ctx:OxV)
framework/include/media/MediaRecorder.h:169: ERROR: [SPC_M_OPR] spaces required around that '<' (ctx:VxV)
framework/include/media/MediaRecorder.h:169: ERROR: [SPC_M_OPR] spaces required around that ':' (ctx:VxO)
framework/include/media/MediaRecorder.h:169: ERROR: [SPC_M_OPR] spaces required around that ':' (ctx:OxV)
framework/include/media/MediaRecorder.h:169: ERROR: [SPC_M_OPR] spaces required around that '>' (ctx:VxW)
framework/include/media/MediaRecorder.h:180: ERROR: [SPC_M_OPR] spaces required around that ':' (ctx:VxO)
framework/include/media/MediaRecorder.h:180: ERROR: [SPC_M_OPR] spaces required around that ':' (ctx:OxV)
framework/include/media/MediaRecorder.h:180: ERROR: [SPC_M_OPR] spaces required around that '<' (ctx:VxV)
framework/include/media/MediaRecorder.h:180: ERROR: [SPC_M_OPR] spaces required around that '>' (ctx:VxW)
framework/include/media/MediaRecorder.h:195: ERROR: [SPC_M_OPR] spaces required around that ':' (ctx:OxV)
framework/include/media/MediaRecorder.h:195: ERROR: [SPC_M_OPR] spaces required around that '<' (ctx:VxV)
framework/include/media/MediaRecorder.h:195: ERROR: [SPC_M_OPR] spaces required around that '>' (ctx:VxW)
framework/include/media/OutputDataSource.h:43: ERROR: [SPC_M_OPR] spaces prohibited around that ':' (ctx:WxW)
framework/include/media/OutputDataSource.h:73: ERROR: [SPC_M_OPR] spaces required around that '=' (ctx:VxV)
framework/include/media/OutputDataSource.h:82: ERROR: [SPC_M_OPR] spaces required around that ':' (ctx:OxV)
framework/include/media/OutputDataSource.h:82: ERROR: [SPC_M_OPR] spaces required around that '<' (ctx:VxV)
framework/include/media/OutputDataSource.h:82: ERROR: [SPC_M_OPR] spaces required around that '>' (ctx:VxW)
framework/include/media/OutputDataSource.h:83: ERROR: [SPC_M_OPR] spaces required around that ':' (ctx:VxO)
framework/include/media/OutputDataSource.h:83: ERROR: [SPC_M_OPR] spaces required around that ':' (ctx:OxV)
framework/include/media/OutputDataSource.h:83: ERROR: [SPC_M_OPR] spaces required around that '<' (ctx:VxV)
framework/include/media/OutputDataSource.h:83: ERROR: [SPC_M_OPR] spaces required around that '>' (ctx:VxW)
framework/include/media/OutputDataSource.h:89: ERROR: [SPC_M_OPR] spaces required around that ':' (ctx:OxV)
framework/include/media/OutputDataSource.h:89: ERROR: [SPC_M_OPR] spaces required around that '<' (ctx:VxV)
framework/include/media/OutputDataSource.h:89: ERROR: [SPC_M_OPR] spaces required around that '>' (ctx:VxW)
framework/include/media/SocketOutputDataSource.h:46: ERROR: [SPC_M_OPR] spaces prohibited around that ':' (ctx:WxW)
framework/include/media/SocketOutputDataSource.h:63: ERROR: [SPC_M_OPR] spaces required around that ':' (ctx:OxV)
framework/include/media/SocketOutputDataSource.h:75: ERROR: [SPC_M_OPR] spaces required around that ':' (ctx:OxV)
framework/include/media/SocketOutputDataSource.h:87: ERROR: [SPC_M_OPR] spaces required around that '=' (ctx:VxV)
framework/include/media/SocketOutputDataSource.h:129: ERROR: [SPC_M_OPR] spaces required around that ':' (ctx:OxV)
framework/src/media/MediaPlayerImpl.h:51: ERROR: [SPC_M_OPR] spaces prohibited around that ':' (ctx:WxW)
framework/src/media/MediaPlayerImpl.h:51: ERROR: [SPC_M_OPR] spaces required around that ':' (ctx:VxO)
framework/src/media/MediaPlayerImpl.h:51: ERROR: [SPC_M_OPR] spaces required around that ':' (ctx:OxV)
framework/src/media/MediaPlayerImpl.h:51: ERROR: [SPC_M_OPR] spaces required around that '<' (ctx:VxV)
framework/src/media/MediaPlayerImpl.h:51: ERROR: [SPC_M_OPR] spaces required around that '>' (ctx:VxE)
framework/src/media/MediaPlayerImpl.h:68: ERROR: [SPC_M_OPR] spaces required around that ':' (ctx:VxO)
framework/src/media/MediaPlayerImpl.h:68: ERROR: [SPC_M_OPR] spaces required around that ':' (ctx:OxV)
framework/src/media/MediaPlayerImpl.h:68: ERROR: [SPC_M_OPR] spaces required around that '<' (ctx:VxV)
framework/src/media/MediaPlayerImpl.h:68: ERROR: [SPC_M_OPR] spaces required around that ':' (ctx:VxO)
framework/src/media/MediaPlayerImpl.h:68: ERROR: [SPC_M_OPR] spaces required around that ':' (ctx:OxV)
framework/src/media/MediaPlayerImpl.h:68: ERROR: [SPC_M_OPR] spaces required around that '>' (ctx:VxB)
framework/src/media/MediaPlayerImpl.h:69: ERROR: [SPC_M_OPR] spaces required around that ':' (ctx:VxO)
framework/src/media/MediaPlayerImpl.h:69: ERROR: [SPC_M_OPR] spaces required around that ':' (ctx:OxV)
framework/src/media/MediaPlayerImpl.h:69: ERROR: [SPC_M_OPR] spaces required around that '<' (ctx:VxV)
framework/src/media/MediaPlayerImpl.h:69: ERROR: [SPC_M_OPR] spaces required around that '>' (ctx:VxB)
framework/src/media/MediaPlayerImpl.h:88: ERROR: [SPC_M_OPR] spaces required around that ':' (ctx:VxO)
framework/src/media/MediaPlayerImpl.h:88: ERROR: [SPC_M_OPR] spaces required around that ':' (ctx:OxV)
framework/src/media/MediaPlayerImpl.h:88: ERROR: [SPC_M_OPR] spaces required around that '<' (ctx:VxV)
framework/src/media/MediaPlayerImpl.h:88: ERROR: [SPC_M_OPR] spaces required around that '>' (ctx:VxW)
framework/src/media/MediaPlayerImpl.h:89: ERROR: [SPC_M_OPR] spaces required around that ':' (ctx:VxO)
framework/src/media/MediaPlayerImpl.h:89: ERROR: [SPC_M_OPR] spaces required around that ':' (ctx:OxV)
framework/src/media/MediaPlayerImpl.h:89: ERROR: [SPC_M_OPR] spaces required around that '<' (ctx:VxV)
framework/src/media/MediaPlayerImpl.h:89: ERROR: [SPC_M_OPR] spaces required around that ':' (ctx:VxO)
framework/src/media/MediaPlayerImpl.h:89: ERROR: [SPC_M_OPR] spaces required around that ':' (ctx:OxV)
framework/src/media/MediaPlayerImpl.h:89: ERROR: [SPC_M_OPR] spaces required around that '>' (ctx:VxW)
framework/src/media/MediaPlayerImpl.h:92: ERROR: [SPC_M_OPR] spaces required around that ':' (ctx:OxV)
framework/src/media/MediaPlayerImpl.h:92: ERROR: [SPC_M_OPR] spaces required around that '<' (ctx:VxV)
framework/src/media/MediaPlayerImpl.h:92: ERROR: [SPC_M_OPR] spaces required around that '>' (ctx:VxW)
framework/src/media/MediaPlayerImpl.h:95: ERROR: [SPC_M_OPR] spaces required around that ':' (ctx:OxV)
framework/src/media/MediaPlayerImpl.h:96: ERROR: [SPC_M_OPR] spaces required around that ':' (ctx:OxV)
framework/src/media/MediaPlayerImpl.h:97: ERROR: [SPC_M_OPR] spaces required around that ':' (ctx:OxV)
framework/src/media/MediaPlayerImpl.h:97: ERROR: [SPC_M_OPR] spaces required around that '<' (ctx:VxV)
framework/src/media/MediaPlayerImpl.h:97: ERROR: [SPC_M_OPR] spaces required around that '>' (ctx:VxW)
framework/src/media/MediaPlayerImpl.h:98: ERROR: [SPC_M_OPR] spaces required around that ':' (ctx:OxV)
framework/src/media/MediaPlayerImpl.h:98: ERROR: [SPC_M_OPR] spaces required around that '<' (ctx:VxV)
framework/src/media/MediaPlayerImpl.h:98: ERROR: [SPC_M_OPR] spaces required around that ':' (ctx:VxO)
framework/src/media/MediaPlayerImpl.h:98: ERROR: [SPC_M_OPR] spaces required around that ':' (ctx:OxV)
framework/src/media/MediaPlayerImpl.h:98: ERROR: [SPC_M_OPR] spaces required around that '>' (ctx:VxW)
framework/src/media/MediaQueue.h:35: ERROR: [SPC_M_OPR] spaces required around that '<' (ctx:WxV)
framework/src/media/MediaQueue.h:35: ERROR: [SPC_M_OPR] spaces required around that '>' (ctx:VxW)
framework/src/media/MediaQueue.h:35: ERROR: [SPC_M_OPR] spaces required around that '&&' (ctx:WxV)
framework/src/media/MediaQueue.h:35: ERROR: [SPC_M_OPR] spaces required around that '&&' (ctx:WxV)
framework/src/media/MediaQueue.h:37: ERROR: [SPC_M_OPR] spaces required around that ':' (ctx:OxV)
framework/src/media/MediaQueue.h:37: ERROR: [SPC_M_OPR] spaces required around that '<' (ctx:VxV)
framework/src/media/MediaQueue.h:37: ERROR: [SPC_M_OPR] spaces required around that ':' (ctx:VxO)
framework/src/media/MediaQueue.h:37: ERROR: [SPC_M_OPR] spaces required around that ':' (ctx:OxV)
framework/src/media/MediaQueue.h:37: ERROR: [SPC_M_OPR] spaces required around that '>' (ctx:VxW)
framework/src/media/MediaQueue.h:38: ERROR: [SPC_M_OPR] spaces required around that ':' (ctx:OxV)
framework/src/media/MediaQueue.h:38: ERROR: [SPC_M_OPR] spaces required around that '<' (ctx:VxV)
framework/src/media/MediaQueue.h:38: ERROR: [SPC_M_OPR] spaces required around that '>' (ctx:VxW)
framework/src/media/MediaQueue.h:38: ERROR: [SPC_M_OPR] spaces required around that ':' (ctx:VxO)
framework/src/media/MediaQueue.h:38: ERROR: [SPC_M_OPR] spaces required around that ':' (ctx:OxV)
framework/src/media/MediaQueue.h:38: ERROR: [SPC_M_OPR] spaces required around that ':' (ctx:VxO)
framework/src/media/MediaQueue.h:38: ERROR: [SPC_M_OPR] spaces required around that ':' (ctx:OxV)
framework/src/media/MediaQueue.h:38: ERROR: [SPC_M_OPR] spaces required around that '<' (ctx:VxV)
framework/src/media/MediaQueue.h:38: ERROR: [SPC_M_OPR] spaces required around that '>' (ctx:VxV)
framework/src/media/MediaQueue.h:38: ERROR: [SPC_M_OPR] spaces required around that ':' (ctx:VxO)
framework/src/media/MediaQueue.h:38: ERROR: [SPC_M_OPR] spaces required around that ':' (ctx:OxV)
framework/src/media/MediaQueue.h:38: ERROR: [SPC_M_OPR] spaces required around that '<' (ctx:VxV)
framework/src/media/MediaQueue.h:38: ERROR: [SPC_M_OPR] spaces required around that '>' (ctx:VxV)
framework/src/media/MediaQueue.h:42: ERROR: [SPC_M_OPR] spaces required around that ':' (ctx:OxV)
framework/src/media/MediaQueue.h:42: ERROR: [SPC_M_OPR] spaces required around that '<' (ctx:VxV)
framework/src/media/MediaQueue.h:42: ERROR: [SPC_M_OPR] spaces required around that '>' (ctx:VxW)
framework/src/media/MediaQueue.h:46: ERROR: [SPC_M_OPR] spaces required around that ':' (ctx:OxV)
framework/src/media/MediaQueue.h:46: ERROR: [SPC_M_OPR] spaces required around that '<' (ctx:VxV)
framework/src/media/MediaQueue.h:46: ERROR: [SPC_M_OPR] spaces required around that ':' (ctx:VxO)
framework/src/media/MediaQueue.h:46: ERROR: [SPC_M_OPR] spaces required around that ':' (ctx:OxV)
framework/src/media/MediaQueue.h:46: ERROR: [SPC_M_OPR] spaces required around that '<' (ctx:VxV)
framework/src/media/MediaQueue.h:46: ERROR: [SPC_M_OPR] need consistent spacing around '>>' (ctx:VxW)
framework/src/media/MediaQueue.h:47: ERROR: [SPC_M_OPR] spaces required around that ':' (ctx:OxV)
framework/src/media/MediaQueue.h:48: ERROR: [SPC_M_OPR] spaces required around that ':' (ctx:OxV)
framework/src/media/MediaRecorderImpl.h:64: ERROR: [SPC_M_OPR] spaces prohibited around that ':' (ctx:WxW)
framework/src/media/MediaRecorderImpl.h:64: ERROR: [SPC_M_OPR] spaces required around that '<' (ctx:VxV)
framework/src/media/MediaRecorderImpl.h:64: ERROR: [SPC_M_OPR] spaces required around that '>' (ctx:VxE)
framework/src/media/MediaRecorderImpl.h:81: ERROR: [SPC_M_OPR] spaces required around that ':' (ctx:VxO)
framework/src/media/MediaRecorderImpl.h:81: ERROR: [SPC_M_OPR] spaces required around that ':' (ctx:OxV)
framework/src/media/MediaRecorderImpl.h:81: ERROR: [SPC_M_OPR] spaces required around that '<' (ctx:VxV)
framework/src/media/MediaRecorderImpl.h:81: ERROR: [SPC_M_OPR] spaces required around that ':' (ctx:VxO)
framework/src/media/MediaRecorderImpl.h:81: ERROR: [SPC_M_OPR] spaces required around that ':' (ctx:OxV)
framework/src/media/MediaRecorderImpl.h:81: ERROR: [SPC_M_OPR] spaces required around that '>' (ctx:VxW)
framework/src/media/MediaRecorderImpl.h:83: ERROR: [SPC_M_OPR] spaces required around that ':' (ctx:VxO)
framework/src/media/MediaRecorderImpl.h:83: ERROR: [SPC_M_OPR] spaces required around that ':' (ctx:OxV)
framework/src/media/MediaRecorderImpl.h:83: ERROR: [SPC_M_OPR] spaces required around that '<' (ctx:VxV)
framework/src/media/MediaRecorderImpl.h:83: ERROR: [SPC_M_OPR] spaces required around that '>' (ctx:VxW)
framework/src/media/MediaRecorderImpl.h:99: ERROR: [SPC_M_OPR] spaces required around that ':' (ctx:VxO)
framework/src/media/MediaRecorderImpl.h:99: ERROR: [SPC_M_OPR] spaces required around that ':' (ctx:OxV)
framework/src/media/MediaRecorderImpl.h:99: ERROR: [SPC_M_OPR] spaces required around that '<' (ctx:VxV)
framework/src/media/MediaRecorderImpl.h:99: ERROR: [SPC_M_OPR] spaces required around that '>' (ctx:VxW)
framework/src/media/MediaRecorderImpl.h:100: ERROR: [SPC_M_OPR] spaces required around that ':' (ctx:VxO)
framework/src/media/MediaRecorderImpl.h:100: ERROR: [SPC_M_OPR] spaces required around that ':' (ctx:OxV)
framework/src/media/MediaRecorderImpl.h:100: ERROR: [SPC_M_OPR] spaces required around that '<' (ctx:VxV)
framework/src/media/MediaRecorderImpl.h:100: ERROR: [SPC_M_OPR] spaces required around that ':' (ctx:VxO)
framework/src/media/MediaRecorderImpl.h:100: ERROR: [SPC_M_OPR] spaces required around that ':' (ctx:OxV)
framework/src/media/MediaRecorderImpl.h:100: ERROR: [SPC_M_OPR] spaces required around that '>' (ctx:VxW)
framework/src/media/MediaRecorderImpl.h:104: ERROR: [SPC_M_OPR] spaces required around that ':' (ctx:OxV)
framework/src/media/MediaRecorderImpl.h:104: ERROR: [SPC_M_OPR] spaces required around that '<' (ctx:VxV)
framework/src/media/MediaRecorderImpl.h:104: ERROR: [SPC_M_OPR] spaces required around that '>' (ctx:VxW)
framework/src/media/MediaRecorderImpl.h:105: ERROR: [SPC_M_OPR] spaces required around that ':' (ctx:OxV)
framework/src/media/MediaRecorderImpl.h:105: ERROR: [SPC_M_OPR] spaces required around that '<' (ctx:VxV)
framework/src/media/MediaRecorderImpl.h:105: ERROR: [SPC_M_OPR] spaces required around that ':' (ctx:VxO)
framework/src/media/MediaRecorderImpl.h:105: ERROR: [SPC_M_OPR] spaces required around that ':' (ctx:OxV)
framework/src/media/MediaRecorderImpl.h:105: ERROR: [SPC_M_OPR] spaces required around that '>' (ctx:VxW)
framework/src/media/MediaRecorderImpl.h:106: ERROR: [SPC_M_OPR] spaces required around that ':' (ctx:OxV)
framework/src/media/MediaRecorderImpl.h:106: ERROR: [SPC_M_OPR] spaces required around that '<' (ctx:VxV)
framework/src/media/MediaRecorderImpl.h:106: ERROR: [SPC_M_OPR] spaces required around that '>' (ctx:VxW)
framework/src/media/MediaRecorderImpl.h:111: ERROR: [SPC_M_OPR] spaces required around that ':' (ctx:OxV)
framework/src/media/MediaWorker.h:37: ERROR: [SPC_M_OPR] spaces required around that '<' (ctx:WxV)
framework/src/media/MediaWorker.h:37: ERROR: [SPC_M_OPR] spaces required around that '>' (ctx:VxW)
framework/src/media/MediaWorker.h:37: ERROR: [SPC_M_OPR] spaces required around that '&&' (ctx:WxV)
framework/src/media/MediaWorker.h:37: ERROR: [SPC_M_OPR] spaces required around that '&&' (ctx:WxV)
framework/src/media/MediaWorker.h:41: ERROR: [SPC_M_OPR] spaces required around that ':' (ctx:OxV)
framework/src/media/MediaWorker.h:41: ERROR: [SPC_M_OPR] spaces required around that '<' (ctx:VxV)
framework/src/media/MediaWorker.h:41: ERROR: [SPC_M_OPR] spaces required around that '>' (ctx:VxW)
framework/src/media/MediaWorker.h:53: ERROR: [SPC_M_OPR] spaces required around that ':' (ctx:OxV)
framework/src/media/MediaWorker.h:53: ERROR: [SPC_M_OPR] spaces required around that '<' (ctx:VxV)
framework/src/media/MediaWorker.h:53: ERROR: [SPC_M_OPR] spaces required around that '>' (ctx:VxW)
framework/src/media/MediaWorker.h:56: ERROR: [SPC_M_OPR] spaces required around that ':' (ctx:OxV)
framework/src/media/PlayerObserverWorker.h:36: ERROR: [SPC_M_OPR] spaces prohibited around that ':' (ctx:WxW)
framework/src/media/RecorderObserverWorker.h:38: ERROR: [SPC_M_OPR] spaces prohibited around that ':' (ctx:WxW)
framework/src/media/RecorderWorker.h:40: ERROR: [SPC_M_OPR] spaces prohibited around that ':' (ctx:WxW)
framework/src/media/RecorderWorker.h:45: ERROR: [SPC_M_OPR] spaces required around that ':' (ctx:OxV)
framework/src/media/RecorderWorker.h:45: ERROR: [SPC_M_OPR] spaces required around that '<' (ctx:VxV)
framework/src/media/RecorderWorker.h:45: ERROR: [SPC_M_OPR] spaces required around that '>' (ctx:VxW)
framework/src/media/RecorderWorker.h:46: ERROR: [SPC_M_OPR] spaces required around that ':' (ctx:VxO)
framework/src/media/RecorderWorker.h:46: ERROR: [SPC_M_OPR] spaces required around that ':' (ctx:OxV)
framework/src/media/RecorderWorker.h:46: ERROR: [SPC_M_OPR] spaces required around that '<' (ctx:VxV)
framework/src/media/RecorderWorker.h:46: ERROR: [SPC_M_OPR] spaces required around that '>' (ctx:VxW)
framework/src/media/RecorderWorker.h:54: ERROR: [SPC_M_OPR] spaces required around that ':' (ctx:OxV)
framework/src/media/RecorderWorker.h:54: ERROR: [SPC_M_OPR] spaces required around that '<' (ctx:VxV)
framework/src/media/RecorderWorker.h:54: ERROR: [SPC_M_OPR] spaces required around that '>' (ctx:VxW)
framework/src/media/audio/audio_manager.c:134: ERROR: [SPC_M_SEP] space required after that close brace '}'
framework/src/media/utils/MediaUtils.h:44: ERROR: [SPC_M_OPR] spaces required around that ':' (ctx:VxO)
framework/src/media/utils/MediaUtils.h:44: ERROR: [SPC_M_OPR] spaces required around that ':' (ctx:OxV)
framework/src/media/utils/MediaUtils.h:51: ERROR: [SPC_M_OPR] spaces required around that ':' (ctx:VxO)
framework/src/media/utils/MediaUtils.h:51: ERROR: [SPC_M_OPR] spaces required around that ':' (ctx:OxV)
framework/src/media/utils/MediaUtils.h:59: ERROR: [SPC_M_OPR] spaces required around that ':' (ctx:VxO)
framework/src/media/utils/MediaUtils.h:59: ERROR: [SPC_M_OPR] spaces required around that ':' (ctx:OxV)

**Comment by junmin-kim** [comment]
@Taejun-Kwon Could you review this PR?

File: framework/src/media/Decoder.h [hunk 1]
```diff
@@ -40,7 +40,7 @@ class Decoder
 
 private:
 #ifdef CONFIG_AUDIO_CODEC
-	//static int _configFunc(void *user_data, int audio_type, void *dec_ext);
+	// static int _configFunc(void *user_data, int audio_type, void *dec_ext);
```
Comment (chanijjani) [inline framework/src/media/Decoder.h]: How about just deleting this line?
Is it just for backup? Then how about adding ToDo here..

File: framework/include/media/MediaPlayer.h [hunk 2]
```diff
@@ -50,11 +50,11 @@ typedef enum player_result_e {
 class MediaPlayerImpl;
 
 /**
- * @class 
+ * @class
  * @brief This class implements the MediaPlayer capability agent.
  * @details @b #include <media/MediaPlayer.h>
  * @since TizenRT v2.0 PRE
-*/
+ */
```
Comment (chanijjani) [inline framework/include/media/MediaPlayer.h]: Is space added?

File: framework/src/media/MediaRecorderImpl.cpp [hunk 3]
```diff
@@ -129,13 +129,13 @@ recorder_result_t MediaRecorderImpl::prepare()
...
-void MediaRecorderImpl::prepareRecorder(recorder_result_t& ret)
+void MediaRecorderImpl::prepareRecorder(recorder_result_t &ret)
 {
 	medvdbg("prepareRecorder mCurState : %d\n", (recorder_state_t)mCurState);
 
 	if (mCurState != RECORDER_STATE_IDLE || mOutputDataSource == nullptr) {
-		meddbg("prepare Failed mCurState: %d mOutputDataSource : %s\n", (recorder_state_t)mCurState, \
-			(mOutputDataSource == nullptr ? "nullptr" : "not null"));
+		meddbg("prepare Failed mCurState: %d mOutputDataSource : %s\n", (recorder_state_t)mCurState,
+			   (mOutputDataSource == nullptr ? "nullptr" : "not null"));
```
Comment (chanijjani) [inline framework/src/media/MediaRecorderImpl.cpp]: Is it correct to add a tab(?)

File: framework/src/media/audio/audio_manager.c [hunk 4]
```diff
@@ -70,21 +70,16 @@
...
 	AUDIO_CARD_IDLE = 1,
 	AUDIO_CARD_READY,
 	AUDIO_CARD_RUNNING,
 	AUDIO_CARD_PAUSE
 };
 
-enum manufacturer_type_e {
-	AUDIO_CARD_REALTEK = 0
-};
+enum manufacturer_type_e { AUDIO_CARD_REALTEK = 0 };
```
Comment (chanijjani) [inline framework/src/media/audio/audio_manager.c]: I already applied our cformatter.sh to audio_manager.c and audio_manager.h several times.
The existing cformatter is wrong...?

File: framework/src/media/audio/audio_manager.h [hunk 5]
```diff
@@ -369,6 +369,6 @@ audio_manager_result_t set_input_audio_volume(uint8_t volume);
 audio_manager_result_t set_output_audio_volume(uint8_t volume);
 
 #if defined(__cplusplus)
-}								/* extern "C" */
+} /* extern "C" */
```
Comment (chanijjani) [inline framework/src/media/audio/audio_manager.h]: I already applied our cformatter.sh to audio_manager.c and audio_manager.h several times.
The existing cformatter is wrong...?

**Comment by chanijjani** [comment]
Also, If this is merged, it needs some PRs related to our media framework to be uploaded again after resolving conflicts.
So please pend the merging of this PR for one or two days...

**Pull request closed without merging**